.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
```

Comment:
Can we do same here, `labels: feature-request, triage me`

Reply:
Sure, done.